### PR TITLE
images.js added to route upload images.

### PR DESCRIPTION
**Pull request by teamagam1**
images.js added to route upload images.

the image is uploaded to storage and a message with it's URL is saved in our messagesDB..

the http post should contain two fields:
1. key: image, value: file
2. key: message, value: JSON string representing image message

returns message image with url to the stored file.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/teamagam/GGMessaging/pull/5" target="_blank">ReviewNinja

File: config/config.json [hunk 1]
```diff
@@ -6,5 +6,9 @@
     "host": "localhost",
     "port": 27017
   },
+  "storage": {
+    "host_name": "cv2-pc",
```
Comment (teamagam3) [inline config/config.json:10]: Should it be like the property above?
We should keep it consistent !fix

File: routes/common.js [hunk 2]
```diff
@@ -4,7 +4,36 @@
...
+/**
+ * Async save new message to Mongo "Message" collection
+ * @param newMsg - new message to be saved in the mongoDB. the message should be constructed to match message Schema.
+ * @param onFailure - callback to be used in-case save fails
+ * @param onSuccess - callback to be used in-case a successful result is returned
+ */
+function saveNewMessageToMongoDB(newMsg, onFailure, onSuccess){
+    if (newMsg.createdAt) {
+        var err = new Error("Given message shouldn't contain createdAt path!");
+        //return is used to finish function's execution
```
Comment (teamagam3) [inline routes/common.js:18]: A comment should have blank line above it.
If the line above it "new block", there is no need to so.
For example, if there is a comment after "if" statement, we shouldn't add empty line.

File: routes/images.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+var common = require('../routes/common');
+
+
+//path to storageDB
+var devConnectionStorage = "http://" + config.storage.host_name + ":" + config.storage.port;
+var connectionStorage = process.env.MONGO_CON_STRING || devConnectionStorage;
+var urlStorage = connectionStorage + "/storage";
+
+var upload = multer({
+    inMemory: true
```
Comment (teamagam3) [inline routes/images.js:15]: A config value would be better

File: config/config.json (hunk 1 above)
Comment (teamagam1) [inline config/config.json:10]: !fixed

**Pull request merged**